_posts/en/2021-12-10-iitb-diagnostic-study.md

```diff
@@ -0,0 +1,139 @@
+By doing so, we wanted to collect information that would inform how we can invest in our internal capabilities to improve IITB service delivery, and to develop upcoming plans and strategies.
+You might ask, ‘"why would you do this?”’ as other reports have been completed: [Price Waterhouse Coopers (PwC) study](https://www.pwc.com/ca/en/today-s-issues/upskilling/canadian-pulse-survey.html) conducted in 2020, [the Gartner iScore report](https://www.gartner.com/en).
+
+After studying both reports, each of which took a top-down approach to evaluate the organization, speaking with those at the director-general level and above to understand the organization's strategic needs, there remained a need to better understand the operational environment within IITB.
+In addition, there was a need to develop IITB’s internal knowledge and skills around conducting internal studies.
+
+We strongly believe that directors and their teams are the ones responsible for delivering the IT services that enable ESDC to deliver its mandate to better Canadians’ lives; they are the ones who understand the barriers that impede their teams from performing at their best and know what empowers their teams to consistently perform as well as they do.
+For this reason, we chose to conduct this study using a bottom-up approach, interviewing directors with input from their subordinates.
+We wanted directors to speak up and to be heard.
+In effect, this study was created by IITB for IITB.  
+
+## Interview preparation
+
+The interview questions were designed in collaboration with HR, Public Opinion Research, and data from the Public Service Employee Survey (PSES).
+To prepare for this series of interviews, we consulted with the departmental Values and Ethics team to ensure we meet all Values and Ethics requirements.
+Before conducting the interviews, the questions were sent in advance to give directors time to reflect, ensure the best use of their time, and to allow them to prepare.
+
+We had several goals in mind when we approached the interviews.
+First, to provide a safe space for directors in order for them to feel comfortable talking to us.
+During the interview we assured the directors that all information would be collected anonymously, and no names would be recorded.
+Any information that may identify an employee or team was genericized so that it could not easily be traced. Additionally, we wanted to make the interviews feel like a conversation, rather than an interrogation.
+The intention of the discussion was to listen to each story and understand what is working, and what is not.
+Conducting the interviews in a non-confrontational way, where directors would be comfortable sharing their honest views with us was very important.
+Asking a director to be transparent and objective in their appraisal of the branch when they are stressed about a deadline, or weary of the interviewers (who in many cases were strangers to the directors), would not have been the right environment.
+
```

Comment:
We can share the survey if you wish.  This opens up the transparency and allows offer departments to use it if the want.